src/main/java/de/feu/massim22/group3/agents/DesireHandler.java

```diff
@@ -0,0 +1,85 @@
+package de.feu.massim22.group3.agents;
+
+import java.util.LinkedList;
+
+import de.feu.massim22.group3.agents.Desires.*;
+import de.feu.massim22.group3.agents.Desires.SubDesires.*;
+import de.feu.massim22.group3.utils.logging.AgentLogger;
+
+public class DesireHandler {
+
+    private BdiAgent agent;
+    private Desire currentTeamGoal = null;
+    private LinkedList<Desire> highPrioAgentGoals = new LinkedList<Desire>();
+    private LinkedList<Desire> lowPrioAgentGoals = new LinkedList<Desire>();
+
+    public DesireHandler(BdiAgent agent) {
+        this.agent = agent;
+        setHighPrioAgentGoals();
+        setLowPrioAgentGoals();
+        setTeamGoal(Desires.PROCESS_TASK);
+    }
+
+    public void setTeamGoal(Desires newDesire) {
```

Comment:
Es gibt teilweise mehrere Tasks, glaube wir benötigen hier eine Strategie zur Filterung und Entscheidung welche Tasks in die Desires aufgenommen werden. (Können wir aber vorerst so lassen)